packages/http/fetch/src/observabilityOptions.ts

```diff
@@ -0,0 +1,39 @@
+import { RequestOption } from "@microsoft/kiota-abstractions";
+
+/** Holds the tracing, metrics and logging configuration for the request adapter */
+export interface ObservabilityOptions {
+	includeEUIIAttributes: boolean;
+}
+/** Internal interface not meant to be used externally and designed to facilitate JSON usage of the other interface. */
+export interface ObservabilityOptionsInternal {
```

Comment:
Remove the export here if not meant to be used outside this file scope

Reply:
done